tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -411,3 +411,10 @@ test_spi_bmp388: bmp388@3a {
 	spi-max-frequency = <0>;
 	int-gpios = <&test_gpio 0 0>;
 };
+
+test_spi_i3g4250d: i3g4250d@3b {
```

Comment:
Need to add this sensor to `tests/drivers/build_all/sensor/prj.conf`

Reply:
Done. Is this build_all test running in the CI? I am wondering why it did not fail without the sensor driver enabled.

Reply:
Yes, the build_all test builds in CI. Now that you enabled the new sensor driver, it's failing. :) Please have a look at the buildkite details

Reply:
Awesome ;)
But I think the fail was not related to my driver. It mentioned some device I never worked with and after a rebase to a more recent main it is gone.